### ISSUE TITLE
Cannot export XML from YDR

### ISSUE THREAD

**Issue by MCT32** [dexyfex/CodeWalker#141]
When I try to export a .ydr file of a building as an XML, the export button is greyed out and I cannot press it and I cannot figure out how to get it working.

**Comment by McP4nth3r** [comment]
a YDR File is a 3D object so its useless and impossible to export it as XML
the other question is now, why you would get the 3d object as XML?

**Comment by Disquse** [comment]
@McP4nth3r you're wrong. CodeWalker can export .#DR files as XMLs. Also there's a quite decent Blender plugin for editing GTAV assets (Sollumz), it uses CW's XMLs.

@MCT32 download the latest version of CodeWalker from the official CW Discord guild.
